aries_cloudagent/protocols/didexchange/v1_0/manager.py

```diff
@@ -104,7 +104,7 @@ async def receive_invitation(
                 auto_accept
                 or (
                     auto_accept is None
-                    and self._session.settings.get("debug.auto_accept_invites")
+                    and self.profile.settings.get("debug.auto_accept_invites")
```

Comment:
Is the same setting available on profile?

Reply:
yes, when a session is created, context is copied from the profile instance